public/data/data-structure-information/heap.json

```diff
@@ -0,0 +1,208 @@
+{
+  "title": "Heap",
+  "description": "",
```

Comment:
add a description for the data structure

Reply:
resolved